lib/tinycolor.dart

```diff
@@ -162,3 +158,9 @@ class TinyColor {
     return _color;
   }
 }
+
+extension on Color {
+  Color copy() {
```

Comment:
please rename to `clone()` (since most other Flutter classes use that naming) and use `=>` here.

Reply:
Another thing in mind is this extension will get exported. It is not private?

Reply:
unnamed extension are private, I will make more explicit and rename the method